Features/Context/Users.php

```diff
@@ -36,6 +36,17 @@ public function iCreateUser(TableNode $users = null)
     }
 
     /**
+     * @When I go to (the) User :username page
+     */
+    public function goToUserPage($username)
+    {
+        $this->iAmOnPage('Users');
+        $this->waitWhileLoading();
+        $this->clickOnTreePath("$username $username");
```

Comment:
But besides that, @joaoinacio is right. 

1. Either the function accepts a username as an argument, in which case it must use the userService to load the user, and get the content's name.
2. or the function accepts a user _content name_, and doesn't have to load the user, but it needs to be specified in the sentence.